views/pools/pools-card/card-trade-info.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, TooltipWrapper, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+import { v4 } from 'uuid';
+
+import QuestionCircleSVG from '../../../components/svg/question.circle';
+import { PoolTradeInfoProps } from './pool-card.types';
+
+const CardTradeInfo: FC<PoolTradeInfoProps> = ({ lines }) => {
+  return (
+    <Box
+      bg="surface"
+      borderRadius="1rem"
+      px="1rem"
+      py="0.5rem"
+      width={['100%', '100%', '100%', '19rem']}
+    >
+      {lines.map((line, index) => (
+        <Box
+          key={v4()}
+          display="flex"
+          justifyContent="space-between"
+          alignItems="flex-start"
+          alignSelf="stretch"
+          py="0.5rem"
+          borderBottom="1px solid"
+          borderColor={lines.length - 1 > index ? '#C6C6CA' : 'transparent'}
+        >
+          <Typography
+            fontSize="0.875rem"
+            lineHeight="1.25rem"
```

Comment:
typographies usually already have the Font sizes specified so you don't need to add them manually unless they are different from the assigned sizes